wildlifecompliance/components/call_email/models.py

```diff
@@ -1,28 +1,38 @@
 from __future__ import unicode_literals
-
+import logging
 from django.db import models
 from django.contrib.postgres.fields.jsonb import JSONField
 from django.db.models import Max
+from django.utils.encoding import python_2_unicode_compatible
 from ledger.accounts.models import EmailUser
 from ledger.licence.models import LicenceType
+from wildlifecompliance.components.main.models import CommunicationsLogEntry, UserAction, Document
 from wildlifecompliance.components.organisations.models import Organisation
 from wildlifecompliance.components.applications.models import Application
 from wildlifecompliance.components.main.models import CommunicationsLogEntry,\
     UserAction, Document
 
 
+logger = logging.getLogger(__name__)
+
+
+def update_compliance_doc_filename(instance, filename):
+    return 'wildlifecompliance/compliance/{}/documents/{}'.format(
+        instance.call_email.id, filename)
+
+
 class Classification(models.Model):
```

Comment:
for CHOICES fields, should set constants for the values and then refer to those constants when defining the choices fields, and anywhere the choice itself is referenced (e.g. default=NAME_CHOICES[0][0] would become Classification.NAME_COMPLAINT, where NAME_COMPLAINT = 'complaint'.

Examples here https://github.com/wilsonc86/ledger/commit/606813eda14abb334c1cd8b14d068804f542fa63#diff-5334bd6ec065eac5179e1eaf68018ebfR1220.